Source/JavaScriptCore/ChangeLog

```diff
@@ -1,3 +1,40 @@
+2022-04-30  Philippe Normand  <philn@igalia.com>
+
+        Web Inspector: Update jsmin to 3.0.1
+        https://bugs.webkit.org/show_bug.cgi?id=239924
+
+        Reviewed by NOBODY (OOPS!).
+
+        Updated jsmin from upstream version, applying a few style changes suggested by
+        check-webkit-style. This new version is Python3-only and also much faster. Old:
+
+        time python /app/webkit/Source/JavaScriptCore/Scripts/jsmin.py < /app/webkit/WebKitBuild/Release/WebInspectorUI/DerivedSources/Main.js > foo.js
+
+        real 1m21.234s
+        user 0m59.580s
+        sys  0m21.253s
+
+        New:
+
+        time python /app/webkit/Source/JavaScriptCore/Scripts/jsmin.py < /app/webkit/WebKitBuild/Release/WebInspectorUI/DerivedSources/Main.js > foo.js
```

Comment:
Presumably `python` on your system is `python3`, correct?

Reply:
Yes, the Flatpak SDK no longer ships Python 2 :)

Reply:
For the sake of Apple folks, can we make this `python3`?

Reply:
In the script shebang you mean? Actually currently there's none at all, I assumed python3 was the default one now, but I can make it explicit in the shebang.

Reply:
I mean in your commit message with the timing.